docs/source/notebooks/model_averaging.ipynb

```diff
@@ -3,7 +3,22 @@
   {
```

Comment:
<p>The only thing I changed here was to maximise over <em>w</em> rather than over <em>N</em> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>